### ISSUE TITLE
Deprecated adapters removed in incorrect version

### ISSUE THREAD

**Issue by chrisaligent** [KnpLabs/Gaufrette#647]
Hi,
Just updated an application and found major breakages, tracked it down to this plugin.

The Adapters have been warning about deprecation since 0.4:
```php
@trigger_error('The ' . __NAMESPACE__ . '\Cache adapter is deprecated since version 0.4 and will be removed in 1.0.', E_USER_DEPRECATED);
```

This is completely fine, but they didn't get removed in `1.0.0`, they were removed in `0.10.0` (tagged just over a month ago). So their removal was completely unexpected and caused a BC break (it's also a SemVer violation). 

Locking down our `composer.json` to the older version until all dependencies have caught up is a simple workaround, but it's worth noting in case others have the same problem. Our `composer.lock` file says we upgrade from `0.9.0` to `0.10.0`.

tldr: `0.1.0 !== 1.0.0`

**Comment by Nyholm** [comment]
Hey. 

You are correct. They were removed earlier than the deprecation message said. However, it is not a SemVer violation. SemVer acts differently before 1.0.

**Comment by chrisaligent** [comment]
Ah yes, that makes sense.
I've locked our application back at `0.9.0` and everything seems to be working well again :+1: 
I will notify the platform developers that they should lock down to `0.9.0` as well, as there are still parts of their codebase which try to use those removed adapters, and I don't think they've realised that the adapters have vanished in the latest release.

**Comment by dmitryuk** [comment]
@chrisaligent please close am issue
